batch2/batch/batch.py

```diff
@@ -567,7 +567,6 @@ def __init__(self, app, id, attributes, callback, userdata, user,
         await self.app['db'].batch.update_record(self.id, cancelled=True, closed=True)
```

Comment:
Github isn't letting me add a comment where I want it to. Can we add a comment to _cancel_jobs and _close_jobs that these are called by the driver and cancel, close, and delete are called by the front end? It took a while for me to figure that out.

Reply:
Did you not want to do this?